.github/workflows/pkg.pr.new.yml

```
@@ -0,0 +1,30 @@
+name: Continuous Publish
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - run: corepack enable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Build
+        run: pnpm -r build
+
+      - run: pnpx pkg-pr-new publish './packages/*'
```

Lets get this more inline with the rest of the workflows in the repo. Also added `--compact`.

```suggestion
jobs:
  build:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v4
      - uses: ./.github/actions/setup
      - run: pnpm buildc all
      - run: pnpx pkg-pr-new publish --compact './packages/*'
```

---

done~